c/include/libsbp/bootload.h

```diff
@@ -84,10 +86,12 @@ typedef struct SBP_ATTR_PACKED {
  * The host initiates the bootloader to jump to the application.
  */
 #define SBP_MSG_BOOTLOADER_JUMP_TO_APP     0x00B1
+#ifndef SBP_DISABLE_VARIABLE_SIZED_ARRAYS
```

Comment:
This isn't a variable sized array?

Reply:
Yeah looks like the generator expression isn't working as intended, will take a look

Reply:
moved the logic into a filter method since it was too hard to suss out in the template. should be fixed now